src/_rules/position.js

```diff
@@ -7,13 +7,19 @@ export const positions = [
 
 export const orders = [
   [
-    /^order-(\d+)$/,
+    /^order-([1-9]|1[0-2])$/,
     ([, d]) => ({ 'order': h.number(d) }),
     { autocomplete: 'order-<num>' },
   ],
   ['order-first', { order: '-9999' }],
   ['order-last', { order: '9999' }],
   ['order-none', { order: '0' }],
+  // matching arbitrary values
+  [
+    /^order-\[(\d*)?]$/,
```

Comment:
Same here I guess? 🙂 
```suggestion
    /^order-\[(\d+)]$/,
```
...or if we want to allow empty brackets we could just remove the question mark in the end `/^order-\[(\d*)]$/` 🙂

Reply:
I don't see why a user should just add an empty bracket, so maybe it's best if I change it so that we don't allow it :)